docs_source_files/content/_index.en.md

```diff
@@ -110,13 +109,23 @@ end
 const {Builder, By, Key, until} = require('selenium-webdriver');
 
 (async function example() {
-    let driver = await new Builder().forBrowser('firefox').build();
+    let driver = await new Builder().forBrowser('chrome').build();
+
     try {
-        await driver.get('https://www.google.com/ncr');
-        await driver.findElement(By.name('q')).sendKeys('cheese', Key.RETURN);
-        let firstResult = await driver.wait(until.elementLocated(By.css('h3>a')),10000);
-        console.log(await firstResult.getText());
-    } finally {
+        // Navigate to Url
+        await driver.get('https://www.google.com');
+
+        // Enter text "cheese" and perform keyboard action "Enter"
+        await driver.findElement(By.name('q')).sendKeys('cheese', Key.ENTER);
+
+        let firstResult = await driver.wait(until.elementLocated(By.css('h3>div')),10000);
+        console.log(await firstResult.getAttribute('textContent'));
+    }
```

Comment:
Let's keep it simple, all the other examples avoid try/catch for the sake of showing a simple example. There is no need to make it more complicated _yet_

Reply:
> Let's keep it simple, all the other examples avoid try/catch for the sake of showing a simple example. There is no need to make it more complicated _yet_

Hi @diemol , removed try catch as suggested.